### DISCUSSION TITLE
Including narrator(composor) in the naming scheme

### DISCUSSION THREAD

**Discussion by bhatton** [seanap/Plex-Audiobook-Guide#98] [General]
Hi all,

I'd really like the narrator to be included in the directory naming scheme to cover multiple productions of the same original text, something like:

audiobooks/author/yyyy - title (narrator)/

The desired outcome is that different productions would appear in plex as distinct titles, or perhaps using Plex's newish {edition-edname} approach if that works out. 

The right mp3tag config has escaped me so far - does anyone have this working successfully already?

Cheers
Ben.

**Comment by seanap** [comment]
In the MP3Tag action you can put the narrator in brackets like this `X:\audiobooks\%albumartist%\%year% - %album% {edition-%composer%}`

The narrator can be a good way to distinguish between most editions but there are edge cases where there are different editions with the same narrator, anything that is Abridged vs unabridged would have the same narrator, and also audible has been releasing 'booktrack' editions that add music but have the same narrator.

Plex's new edition feature is only for movies I think.  I edited an audiobook and I dont see that in the general or tags section.

**Comment by bhatton** [comment]
Thanks seanap - I couldn't work out why the desc and reader txt files weren't going in the right place, then realised the paths were hardcoded.

Just trial and error, but I changed the action to set _DIRECTORY:
`Format value "_FILENAME": %album% (%year%)[ '['%series% %series-part%']']$ifgreater(%_total_files%,1, - pt$num(%track%,2),)`
`Format value "_DIRECTORY": R:\audiobooks\%albumartist%\%series%\%year% - %album% [ '['%series% %series-part%']'] {edition-%composer%}`
Then I changed the exports to use a relative path like this:
`%_folderpath%\desc.txt`
and
`%_folderpath%\reader.txt`

Plex shows the albums side by side, which is good.  The directory name is ignored, and the narrator is shown in the track listing.
